src/bundle/DependencyInjection/Compiler/FieldTypeProcessorPass.php

```diff
@@ -6,12 +6,17 @@
  */
 namespace EzSystems\EzPlatformRestBundle\DependencyInjection\Compiler;
 
+use eZ\Publish\Core\Base\Container\Compiler\TaggedServiceIdsIterator\BackwardCompatibleIterator;
+use LogicException;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
 
 class FieldTypeProcessorPass implements CompilerPassInterface
 {
+    public const FIELD_TYPE_PROCESSOR_SERVICE_TAG = 'ezplatform.field_type.rest.processor';
```

Comment:
Since the tag was never really working we should go with new scheme to avoid adding to a pile of work we already have when it comes to cleanup.
```suggestion
    public const FIELD_TYPE_PROCESSOR_SERVICE_TAG = 'ibexa.rest.field_type.processor';
```